### DISCUSSION TITLE
Cant hide on a screen size and above

### DISCUSSION THREAD

**Discussion by irLinja** [tailwindlabs/tailwindcss#14266] [Help]
Hi, I have this code for desktop and mobile views, I'm expecting these to switch on md size, but the mobile view is always visible.
Am I doing it wrong?
```html

<div class="hidden md:block">
  <p> DESKTOP </p>
</div>

<!-- mobile view -->
<div class="block md:hidden">
  <p> MOBILE </p>
</div>
```
`DESKTOP` hides as soon as the display is under 768
<img width="801" alt="image" src="https://github.com/user-attachments/assets/757e0a9c-11b8-4d1b-9290-f26e7ebcc71d">

`MOBILE` is always visible.
<img width="809" alt="image" src="https://github.com/user-attachments/assets/1b22a308-03a6-44b2-b077-30945da9da97">

**Comment by wongjn** [comment]
Seems to work as expected in [this Tailwind Play](https://play.tailwindcss.com/WFlKyrlPsl) which would lead me to believe it is something particular to your project causing the undesired behavior.

Consider checking:
- Tailwind has compiled since you added the classes.
- The `content` file globs sufficiently cover the file that the code appears in.
- [String concatenation/interpolation is not used to build the class names](https://tailwindcss.com/docs/content-configuration#dynamic-class-names).

Otherwise, consider providing a project that reproduces the unexpected behavior.
